### PR TITLE
[IIIF-472] Remove any at symbols @ from provided handles

### PR DESCRIPTION
**Pull request by hardyoyo**
* drops any at symbols in the provided Slack Handle

### CODE REVIEW

**Comment by ksclarke** [comment]
Why not just remove this once at the point we get it from the user? Then the stored version is correct throughout the rest of the process.

https://github.com/UCLALibrary/bucketeer/blob/master/src/main/java/edu/ucla/library/bucketeer/handlers/LoadCsvHandler.java#L100

**Comment by hardyoyo** [comment]
If I know a verticle might be given bad data, I'd prefer to handle it in the verticle. But, sure, we can also filter at the form handler.

**Comment by ksclarke** [comment]
I'm a little confused by the comment. You have a replace on BatchJobStatusHandler, which is a handler pulling the job from the shared context that is set by the LoadCsvHandler. This is different from a verticle pulling a message off the event queue. The other place you have is on a test where it's sanitizing config data (which I get). I'm not sure of the verticle that you're referencing(?)

File: src/main/java/edu/ucla/library/bucketeer/handlers/LoadCsvHandler.java [hunk 1]
```diff
@@ -97,7 +97,8 @@ public void handle(final RoutingContext aContext) {
         final HttpServerRequest request = aContext.request();
         final MultiMap formAttributes = request.formAttributes();
         final Set<FileUpload> csvUploads = aContext.fileUploads();
-        final String slackHandle = StringUtils.trimToNull(formAttributes.get(Constants.SLACK_HANDLE));
+        final String slackHandle = StringUtils.trimToNull(formAttributes.get(Constants.SLACK_HANDLE)).replace(
```
Comment (ksclarke) [inline src/main/java/edu/ucla/library/bucketeer/handlers/LoadCsvHandler.java]: trimToNull() can return a null, which will give you a NullPointerException on your .replace(). The replace should go down in the `else` where we know the value isn't null.

**Pull request merged**